sdk/clientcore/core/src/main/java/io/clientcore/core/utils/auth/ChallengeHandler.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package io.clientcore.core.util.auth;
+package io.clientcore.core.utils.auth;
```

Comment:
There's a `credentials` package too which is related to auth. Should we move these into that package or create a sub-package `auth.credentials`? Also, we generally don't use abbreviation for auth.

Reply:
This is an interesting question as the classes in `utils.auth` are for handling authorization requests with Basic and Digest auth as well as APIs allowing implementations for other auth forms such as Kerebos. Whereas `credentials` are more forms of authentication supported by services such as API key credentials.

I'm not sure moving these classes would be the correct choice or not.